spec/sample-data/valid/gazette-notice-01.json

```diff
@@ -0,0 +1,46 @@
+{
+  "issue": {
+    "publication": {
+      "publisher": {
+        "name": "Commissioners of Public Works in Ireland",
+        "url": "http://www.opw.ie/"
+      },
+      "jurisdiction_code": "ie",
+      "title": "Iris Oifigiuil",
+      "url": "http://www.irisoifigiuil.ie/"
+    },
+    "id": "1",
+    "title": "Number 1",
+    "edition_id": "2015",
+    "page_range": "1-22",
+    "date_published": "2015-01-02",
+    "url": "http://www.irisoifigiuil.ie/archive/2015/january/Ir020115.PDF",
+    "media_type": "application/pdf",
+    "retrieved_at": "2015-11-12T00:00:00Z"
+  },
+  "id": "3",
+  "title": "IN THE MATTER OF THE COMPANIES ACT 1963-2009 AND IN THE MATTER OF SO CHILDISH LIMITED (In Voluntary Liquidation)",
+  "description": "The voluntary liquidation of So Childsh Limited",
+  "body": "IN THE MATTER OF THE COMPANIES ACT 1963-2009 AND IN THE MATTER OF SO CHILDISH LIMITED (In Voluntary Liquidation)\n\nAt an Extraordinary General Meeting of the above named Company duly convened and held in the Glenroyal Hotel, Glenroyal Centre, Maynooth, Co. Kildare, on the 21st of December, 2011, the following Resolution was duly passed: ...",
+  "classification": [
+    "dissolution"
```

Comment:
Depends how you make an ontological distinction between "ours" and "the gazette's". Naturally, our classification is based on the gazette's. If the gazette goes "APPOINTMENT" and we go "appointment", is "appointment" ours or the gazette's? I think in all cases they are the same, but maybe you can describe a clear distinction.